internal/pkg/cli/env_deploy.go

```diff
@@ -151,6 +157,30 @@ func (o *deployEnvOpts) Execute() error {
 	if err != nil {
 		return fmt.Errorf("upload artifacts for environment %s: %w", o.name, err)
 	}
+	if o.showDiff {
+		output, err := deployer.GenerateCloudFormationTemplate(&deploy.DeployEnvironmentInput{
```

Comment:
What do you think if we move logics within the if statement to another method so that i'll be easier to follow the logic in `Execute`?